package.json

```diff
@@ -48,8 +48,10 @@
   },
   "commitlint": {
     "extends": [
+      "squash-pr",
       "@commitlint/config-conventional"
-    ]
+    ],
+    "rules": { "scope-case": [0] }
```

Comment:
can you elaborate on this?

Reply:
It's from our internal config, so this is to allow for instance `feat(Button): ...` instead of only `feat(button): ...`, which you ran into the other day.

Reply:
nice